### PR TITLE
BUGFIX - Catch error in case of image is not found instead of killing…

### PR DESCRIPTION
**Pull request by sylvainraye**
… the whole import process

### CODE REVIEW

File: Integration/Iterator/AbstractIterator.php [hunk 1]
```diff
@@ -23,7 +23,7 @@ abstract class AbstractIterator implements \Iterator
     /**
      * @var LoggerInterface
      */
-    private $logger;
+    protected $logger;
```
Comment (dxops) [inline Integration/Iterator/AbstractIterator.php]: Makes sense to switch to \Psr\Log\LoggerAwareTrait then

File: Integration/Iterator/ProductIterator.php [hunk 2]
```diff
@@ -6,6 +6,7 @@
 use Akeneo\PimEnterprise\ApiClient\AkeneoPimEnterpriseClientInterface;
 use Gaufrette\Filesystem;
 use Psr\Log\LoggerInterface;
+use Akeneo\Pim\ApiClient\Exception\NotFoundHttpException;
```
Comment (dxops) [inline Integration/Iterator/ProductIterator.php]: Make sure to order import statements please

**Review by dxops — CHANGES_REQUESTED** [review]
Hi, @sylvainraye ,

thank you for your contribution, check my comments below please

File: Integration/Iterator/AbstractIterator.php (hunk 1 above)
Comment (sylvainraye) [reply Integration/Iterator/AbstractIterator.php]: done

File: Integration/Iterator/ProductIterator.php (hunk 2 above)
Comment (sylvainraye) [reply Integration/Iterator/ProductIterator.php]: done

**Pull request merged**